waspc/cli/Command/Db/Migrate.hs

```diff
@@ -36,37 +52,29 @@ migrateSave migrationName = do
         Right () -> waspSaysC "Done."
 
     waspSaysC "Copying migrations folder from Prisma to Wasp project..."
-    copyDbMigDirResult <- liftIO $ copyDbMigrationsDir waspProjectDir genProjectRootDir
-    case copyDbMigDirResult of
+    copyDbMigDirUpResult <- liftIO $ copyDbMigrationsDir CopyMigDirUp waspProjectDir
+                                                         genProjectRootDir
+    case copyDbMigDirUpResult of
         Nothing -> waspSaysC "Done."
         Just err -> throwError $ CommandError $ "Copying migration folder failed: " ++ err
 
     applyAvailableMigrationsAndGenerateClient genProjectRootDir
 
     waspSaysC "All done!"
 
-    where
-        copyDbMigrationsDir
-            :: Path Abs (Dir WaspProjectDir)
-            -> Path Abs (Dir ProjectRootDir)
-            -> IO (Maybe String) -- ^ Possibly contains error message.
-        copyDbMigrationsDir waspProjectDir genProjectRootDir= do
-            let dbMigrationsDirInDbRootDir = SP.fromPathRelDir [P.reldir|migrations|]
-            let dbMigrationsDirSrc = genProjectRootDir </> dbRootDirInProjectRootDir
-                                     </> dbMigrationsDirInDbRootDir
-            let dbMigrationsDirTarget = waspProjectDir </> dbMigrationsDirInDbRootDir
-            
-            ((PathIO.copyDirRecur (SP.toPathAbsDir dbMigrationsDirSrc)
-                                  (SP.toPathAbsDir dbMigrationsDirTarget)) >> return Nothing)
-            `catch` (\e -> return $ Just $ show (e :: P.PathException))
-            `catch` (\e -> return $ Just $ show (e :: IOError))
-
 migrateUp :: Command ()
 migrateUp = do
     waspProjectDir <- findWaspProjectRootDirFromCwd
     let genProjectRootDir = waspProjectDir </> Common.dotWaspDirInWaspProjectDir
                             </> Common.generatedCodeDirInDotWaspDir
 
+    waspSaysC "Copying migrations folder from Wasp to Prisma project..."
```

Comment:
You could have wrapped this piece of code into Command, so you could also reuse it, instead of calling it multiple places with the same stuff around it (output before, error handling later).